app/views/home/index.html.erb

```diff
@@ -1 +1,3 @@
-<H1>The Index Page / Dashboard</h1>
\ No newline at end of file
+<H1>The Index Page / Dashboard</h1>
+
+<%= react_component('New_Project_Button', {link: "projects/new"}) %>
```

Comment:
Ohhhh no way.... since we're mostly in Ruby, if we pass routes around we may not have to keep track of those in React.

but we should use the Rails Routes syntax, use `rake routes` on the command line to know what these are... but it should be `new_projects_path`. so `{link: new_projects_path}`... should work, actually...

Reply:
Thank you. I saw that we were handling routes that way but couldn't figure out how to pass it into a react component. I found that the text worked but I definitely see why we don't want to do that. I'll give your suggestion a go!